src/util/cached-prop-defs.js

```diff
@@ -0,0 +1,83 @@
+//@flow
+import assign from './assign';
+import dashCase from './dash-case';
+import getAllKeys from './get-all-keys';
+
+const CACHED_PROP_DEFS:string = '____skate_propDefs';
```

Comment:
I apologize.
I agree that the SkateJs team is in charge to properly consider what's the best for them. 
I really don't care as long as we have a type checker in place.
I assumed, maybe wrongly, that Flow was an easier and less disrupting choice for the Team because SkateJs is currently built using Babel.

TypeScript adds a level of abstractions. Flow only adds annotations.